lib/dialects/postgres/data-types.js

```diff
@@ -80,6 +80,11 @@ BLOB.prototype.toSql = function() {
   return 'BYTEA';
 };
 
+var GEOMETRY = function() {
```

Comment:
Dialect specific datatypes are only used if a data type is only supported by a single dialect, or if a dialect uses a different type (for example postgres blobs = bytea)

The geometry type is supported by mysql and mssql as well